src/peer_connection.cpp

```diff
@@ -2943,6 +2965,18 @@ namespace libtorrent {
 //			, peer_info_struct(), block_finished.piece_index, block_finished.block_index);
 		picker.mark_as_writing(block_finished, peer_info_struct());
 
+#if 0
+		if (t->info_hash().has_v2())
+		{
+			t->picker().started_hash_job(p.piece);
+			m_disk_thread.async_hash2(t->storage(), p.piece, p.start, {}
+				, [conn = self(), p](piece_index_t, sha256_hash const& h, storage_error const& e)
+			{
+				conn->wrap(&peer_connection::on_hash2_complete, e, p, h);
+			});
+		}
+#endif
```

Comment:
we should decide what to do with this block. I don't quite understand it at first glance, but it looks kind of important

Reply:
I originally intended to compute v2 block hashes incrementally as the blocks where downloaded, but I ended up changing to computing v2 hashes in the same job as the v1 hashes. Eventually I want to support setting deadlines with block granularity which will require downloading leaf hashes and checking individual blocks. This code remains as a reference for when I implement that.

Reply:
would you mind adding a comment in the code explaining this?